.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         run: yarn --frozen-lockfile
 
       - name: Build packages
-        run: yarn build
+        run: yarn build:packages
```

Comment:
Doing this avoids unnecessarily building the website when all we're doing is running the linter.